### DISCUSSION TITLE
Is there a way to have same review simultaneously active on multiple computers?

### DISCUSSION THREAD

**Discussion by mamazebra** [asreview/asreview#1490] [Ideas]
I started an ASREVIEW project on my laptop (typing on it now).  I don't routinely carry laptop from / to home.  I'd like to just fire up my desktop at work and work on same review... but I can't do that (I think) unless I'm routinely exporting?sharing? the review.  which leads to version control headaches.  So having started review on one specific computer (all 37,000 records), I am stuck with I can only work on the review on this same machine unless I constantly export/share at end of every session?

Sometimes I don't know where I'll be working until the next day, so planning ahead isn't simple.  Any other project, I have one work file server that every desktop/laptop can access.  Also at work, we are moving to a hotdesk system for the desktop machines.  Fortunately I have the laptop so I can have one consistent device.

Just seems so weird when everything else is cloud storage to suddenly be limited to local diskspace.

**Comment by sparksfly13** [comment]
I have the same question. it doesn't make sense to me that the ASReview project is only accessible on a single device. what if my laptop crashes? will I lose all my progress?

ideally, i would like to switch between 2 laptops (office/home) too.

**Comment by Rensvandeschoot** [comment]
It is possible to install and run ASReview on a server: https://asreview.readthedocs.io/en/latest/installation.html#server-installation
or in the cloud: https://github.com/asreview/cloud-usage

Would this help?
